linx-impulse/mod.ts

```diff
@@ -0,0 +1,77 @@
+import type {
+  App as A,
+  AppContext as AC,
+  AppMiddlewareContext as AMC,
+} from "deco/mod.ts";
+import { createHttpClient } from "../utils/http.ts";
+import manifest, { Manifest } from "./manifest.gen.ts";
+import { middleware } from "./middleware.ts";
+import { ChaordicAPI } from "./utils/chaordic.ts";
+import { LinxAPI } from "./utils/client.ts";
+import { EventsAPI } from "./utils/events.ts";
+
+export type App = ReturnType<typeof Linx>;
+export type AppContext = AC<App>;
+export type AppMiddlewareContext = AMC<App>;
+
+/** @title LINX Impulse */
+export interface State {
+  /**
+   * @title Api Key
+   */
+  apiKey: string;
+  /**
+   * @title Secret Key
+   */
+  secretKey?: string;
+  /**
+   * @title Origin URL
+   * @description Set the origin URL to authenticate the request if not using the secret key
+   */
+  origin?: string;
+  /**
+   * @title Image CDN URL
+   * @description e.g.: https://{account}.myvtex.com/
```

Comment:
Is myvtex url a good example? Will people use that?

Reply:
It's the same example used in the Linx integration, stores that use linx + vtex have the cdn url like this